ucm/sof-skl_hda_card/HiFi.conf

```diff
@@ -118,6 +122,7 @@ SectionDevice."HDMI1" {
 
 	Value {
 		PlaybackPCM "hw:0,3"
+		PlaybackVolume "PGA7.0 7 Master"
```

Comment:
Do we have a good reason to enable volume control for HDMI?
It can be done in PulseAudio or in the receiver.

Reply:
fine with me, can/will remove it.

Reply:
removed for now.